robocop/checkers/spacing.py

```diff
@@ -54,14 +54,14 @@ def parse_file(self):
                 if empty_lines > 1:
                     self.report("too-many-trailing-blank-lines", lineno=len(self.raw_lines), col=0)
                     return
-            if not empty_lines and not last_line.endswith('\n'):
+            if not empty_lines and not last_line.endswith(('\n','\r')):
```

Comment:
and here (`'/r/n'`)?

Reply:
\r\n also ends with \n, so it is not needed here